apps/meteor/ee/app/livechat-enterprise/server/lib/LivechatEnterprise.js

```diff
@@ -133,18 +133,17 @@ export const LivechatEnterprise = {
 		return LivechatTag.createOrUpdateTag(_id, tagData, tagDepartments);
 	},
 
-	// make async
-	saveSLA(_id, slaData) {
+	async saveSLA(_id, slaData) {
 		check(_id, Match.Maybe(String));
 
 		check(slaData, {
 			name: String,
 			description: Match.Optional(String),
-			dueTimeInMinutes: String,
+			dueTimeInMinutes: Number,
 		});
```

Comment:
This should be already validated by the endpoint, so we can skip double validation here 👀

Reply:
Ouch, I forgot about that. I'd kept it before because the meteor method was calling this same function so these validations would be used by it, however forgot to remove them while removing those methods. Let me do it now